pkg/service/publish_results.go

```diff
@@ -183,12 +182,8 @@ func (mw logmw) PublishResults(ctx context.Context, nodeKey string, results []di
 	defer func(begin time.Time) {
 		resJSON, _ := json.Marshal(results)
 		uuid, _ := uuid.FromContext(ctx)
-		logger := level.Debug(mw.logger)
-		if err != nil {
-			logger = level.Info(mw.logger)
-		}
-		logger.Log(
-			"method", "PublishResults",
+
+		mw.knapsack.Slogger().Log(ctx, levelForError(err), "publish results",
```

Comment:
This is not quite right. -- the first argument should be the `message` from line 189. The `method` should remain a KV pair. (I think true for all of these)

Reply:
Okay, I've added the message val as the `msg` for the logs.

Should we drop the kvp with the `message` key (now it would be `msg`) ... I don't think anything should rely on log output, but I have an urge to leave it in to maintain parity.

Reply:
I think there's no value in duplicating the `message` KV. It's fine that it changed from `message` to `msg`. (I just don't want to have `method` become `msg` which is semantically different)

eg: You should strike `message`.